cmd/configure.go

```diff
@@ -499,6 +503,39 @@ func configureGithub(ctx context.Context, _flags ConfigureGithubFlags) error {
 
 	ctx = events.SetTargetInContext(ctx, workingDir)
 
+	// check if the git repository is a github URI
+	event := shared.CliEvent{}
+	events.EnrichEventWithGitMetadata(ctx, &event)
+
+	// Installing the app is only relevant if we are in a remote linked github repository
+	hasAppAccess := false
+	if event.GitRemoteDefaultOwner != nil && *event.GitRemoteDefaultOwner != "" && event.GitRemoteDefaultRepo != nil && *event.GitRemoteDefaultRepo != "" {
+		hasAppAccess = checkGithubAppAccess(ctx, *event.GitRemoteDefaultOwner, *event.GitRemoteDefaultRepo)
+		if !hasAppAccess {
+			continueAfterInstall := false
+			_, err := charm.NewForm(huh.NewForm(
+				huh.NewGroup(
+					huh.NewSelect[bool]().
+						Title("\n\nSpeakeasy has a github app that can help you setup your repo for SDK generation.\nWould you like to continue after installing the Github App?\n").
```

Comment:
GitHub capitalization consistency

Reply:
set (space) up

Reply:
> Speakeasy has a github app that can help you setup your repo for SDK generation
Wonder if we can make it more clear that installing it will help you _right now_

> Would you like to continue after installing the Github App?
I find this confusing. Maybe just say "Would you like to install it?"

Reply:
Yup can make this adjustments